contracts/RandomNumberConsumer.sol

```diff
@@ -0,0 +1,54 @@
+
+pragma solidity 0.6.6;
+
+import "@chainlink/contracts/src/v0.6/VRFConsumerBase.sol";
+
+contract RandomNumberConsumer is VRFConsumerBase {
+    
+    bytes32 internal keyHash;
+    uint256 internal fee;
+    address public linkTokenAddress;
```

Comment:
VRFConsumerBase already provides the LINK token address in the internal variable `LINK`. This can be removed and `LINK` used instead.